libraries/AP_RangeFinder/AP_RangeFinder.cpp

```diff
@@ -496,6 +497,11 @@ void RangeFinder::detect_instance(uint8_t instance, uint8_t& serial_instance)
             drivers[instance] = new AP_RangeFinder_Lanbao(state[instance], params[instance], serial_instance++);
         }
         break;
+    case Type::LeddarVu8_Serial:
+        if (AP_RangeFinder_Lanbao::detect(serial_instance)) {
```

Comment:
This... doesn't look great :-)

```suggestion
        if (AP_RangeFinder_LeddarVu8::detect(serial_instance)) {
```

Reply:
Thanks for that!  I guess the serial lidars all run the same detect function so it didn't actually show up in testing.  txs!